custom_components/sleep_as_android/sensor.py

```diff
@@ -68,7 +68,38 @@ class SleepAsAndroidSensor(SensorEntity, RestoreEntity):
     """
     _attr_icon = "mdi:sleep"
     _attr_should_poll = False
-    _attr_device_class = f"{DOMAIN}__status"
+    _attr_device_class = SensorDeviceClass.ENUM
+    _attr_options = [
+        "unknown",
+        "sleep_tracking_started",
+        "sleep_tracking_stopped",
+        "sleep_tracking_paused",
+        "sleep_tracking_resumed",
+        "alarm_snooze_clicked",
+        "alarm_snooze_canceled",
+        "time_to_bed_alarm_alert",
+        "alarm_alert_start",
+        "alarm_alert_dismiss",
+        "alarm_skip_next",
+        "show_skip_next_alarm",
+        "rem",
+        "smart_period",
+        "before_smart_period",
+        "lullaby_start",
+        "lullaby_stop",
+        "lullaby_volume_down",
+        "deep_sleep",
+        "light_sleep",
+        "awake",
+        "not_awake",
+        "apnea_alarm",
+        "antisnoring",
+        "sound_event_snore",
+        "sound_event_talk",
+        "sound_event_cough",
+        "sound_event_baby",
+        "sound_event_laugh",
+    ]
```

Comment:
We have same list in triggers https://github.com/IATkachenko/HA-SleepAsAndroid/blob/eeb9ed221a8cdd02c618658487e9843788636a19/custom_components/sleep_as_android/device_trigger.py#L17

The best way is to move it to `const.py`, rename to something like `sleep_tracking_states` and reuse here.
https://en.wikipedia.org/wiki/Don%27t_repeat_yourself
It will keep us from messing something (`before_alarm` is missed in your list).

Reply:
I know DRY, just didn't knew this list already existed :)